cmd/cmdoptions/client.go

```diff
@@ -33,6 +34,8 @@ func (c *ClientOptions) loadTLSConfig() (*tls.Config, error) {
 			return nil, fmt.Errorf("failed to load certs: %s", err)
 		}
 		return &tls.Config{Certificates: []tls.Certificate{cert}}, nil
+	} else if c.EnableTLS {
+		return &tls.Config{}, nil
 	} else if c.ClientKeyPath != "" {
 		return nil, errors.New("got TLS key with no cert")
 	}
```

Comment:
I'd rather still fail if only the key is provided.

```suggestion
	} else if c.ClientKeyPath != "" {
		return nil, errors.New("got TLS key with no cert")
	} else if c.EnableTLS {
		return &tls.Config{}, nil
	}
```